src/lms-ds-loader/edfi_lms_ds_loader/loader_facade.py

```diff
@@ -249,10 +249,10 @@ def run_loader(arguments: MainArguments) -> None:
 
     _load_users(csv_path, db_adapter)
     _load_sections(csv_path, db_adapter)
+    _load_section_associations(csv_path, db_adapter)
```

Comment:
The attendance events have a validation so that it only uploads events for students enrolled in the section. By uploading attendance events first, we were never passing the validation.